example/zipkin/main.go

```diff
@@ -49,7 +49,7 @@ func initTracer(url string) {
 	tp, err := sdktrace.NewProvider(
 		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
 		sdktrace.WithBatcher(exporter,
-			sdktrace.WithScheduleDelayMillis(5),
+			sdktrace.WithBatchTimeout(5),
```

Comment:
5 nanoseconds looks meaningless - I guess old name was confusing and author meant 5 milliseconds (which is still too low).

Reply:
That or they might have wanted this to use a `SpanSyncer` instead. Looks like a common misunderstanding.